presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -816,6 +816,17 @@ public void createSchema(JdbcIdentity identity, String schemaName)
         }
     }
 
+    @Override
+    public void dropSchema(JdbcIdentity identity, String schemaName)
+    {
+        try (Connection connection = connectionFactory.openConnection(identity)) {
+            execute(connection, "DROP SCHEMA " + quoted(schemaName));
```

Comment:
Do we need any access control checks for this?

Reply:
Access control is handled by the engine (see DropSchemaTask)

Reply:
We have a very legacy check for `allowDropTable`. I'll reuse the same one for drop schema.

Reply:
We have a legacy check for `allowDropTable`, which should be removed. Dropping a schema should be safe since databases require the schema to be empty (unless called with `CASCADE` which we don't use).